Tools/glib/generate-pdfjs-gresource-manifest.py

```diff
@@ -23,14 +23,16 @@
 COMPRESSIBLE_EXTENSIONS = ['.html', '.js', '.css', '.svg', '.properties']
 BASE_DIRS = ['pdfjs/', 'pdfjs-extras/']
 
+IGNORE = ['LICENSE',
+          'README.webkit',
+          'web/cmaps/LICENSE',
+          'web/standard_fonts/LICENSE_FOXIT',
+          'web/standard_fonts/LICENSE_LIBERATION']
```

Comment:
I would use a set literal here, but the list is small so in this case using a set instead of a list might not be any advantage. So up to you whether you want to change it or not before landing.

```suggestion
IGNORE = {'LICENSE',
          'README.webkit',
          'web/cmaps/LICENSE',
          'web/standard_fonts/LICENSE_FOXIT',
          'web/standard_fonts/LICENSE_LIBERATION'}
```

Reply:
I've changed all of these lists to sets.